### ISSUE TITLE
Markup schema defines File as required in markup header

### ISSUE THREAD

**Issue by TitouanGisle** [buildingSMART/BCF-XML#253]
Hi !

Currently, according to the `markup.xsd` schema, at least one File element is required in the Header of a Markup file, since its minOccurs attribute defaults to 1 :

```xsd
<xs:complexType name="Header">
	<xs:sequence>
		<xs:element name="File" maxOccurs="unbounded">
                ...
```

Unless I am mistaken, it doesn't seem to be the intended behaviour, since a Topic doesn't need to be related to a File.

It should probably be 
```xsd
<xs:complexType name="Header">
	<xs:sequence>
		<xs:element name="File" minOccurs="0" maxOccurs="unbounded">
                ...
```

**Comment by ykulbak** [comment]
Thank you @TitouanGisle, we'll address the issue in BCF 3.0

**Comment by pasi-paasiala** [comment]
This is fixed together with #260
